src/class/SimpleData.ts

```diff
@@ -32,7 +32,7 @@ import summarize_ from "../methods/summarize.js"
 import saveData_ from "../methods/saveData.js"
 import { SimpleDataItem, Options, defaultOptions } from "../types.js"
 import checkKeys from "../helpers/checkKeys.js"
-
+import logDecorator from "../helpers/logDecorator.js"
```

Comment:
I think it's best practice to avoid using naming variables with their types. For instance, you should not name a list of Potatoes `listOfPotatoes` but instead simply `potatoes`. Especially that here we use `logDecorator` like a simple function. 

Here, I would suggest `logInfos` (since simply `log` is already used)

Reply:
Very good suggestion! Changing it and pushing to main.